nemo/collections/nlp/nm/losses/masked_xentropy_loss.py

```diff
@@ -42,16 +42,16 @@
 from nemo.core.neural_types import LabelsType, LengthsType, LogitsType, LossType, NeuralType
 from nemo.utils.decorators import add_port_docs
 
-__all__ = ['TRADEMaskedCrossEntropy', 'CrossEntropyLoss3D']
+__all__ = ['MaskedXEntropyLoss']
 
 
-class TRADEMaskedCrossEntropy(LossNM):
+class MaskedXEntropyLoss(LossNM):
```

Comment:
Are you sure this should be a lossNM and not a Non-TrainableNM? I do not see any loss calculation in here

Reply:
It actually calculates log loss (binary cross entropy) for a selected element among the all labels. A faster and more over fitting robust version of cross entropy for multi class.

Reply:
I see what you mean now. You are using labels to gather your logits.

Reply:
Thanks for the clarification